ModernFlyouts/AppLifecycle/AppLifecycleManager.cs

```diff
@@ -112,6 +113,6 @@ private static void SignalFirstInstance(string channelName, string[] args)
             Environment.Exit(0);
         }
 
-#endregion
+        #endregion App activation & Single instancing
```

Comment:
Text after `#endregion` will make maintaining the code complex. It would be better the way it was.